### ISSUE TITLE
[Bug] can't craft anything from this mod

### ISSUE THREAD

**Issue by nickademas** [Aton-Kish/quartz-elevator#62]
### To Reproduce

1. Open the game
2. Try to craft elevator

### Environment

- Minecraft: [e.g. 1.20.6]
- Fabric Loader: [0.15.10]
- Fabric API: [0.97.8]

### Logs

Pastebin the log [e.g. `logs/latest.log`] and link it below:

<!-- ```log
or paste the raw log here.
``` --> https://mclo.gs/wONIxRr

### Additional context

Add any other context about the problem here.

**Comment by Aton-Kish** [comment]
Thank you for your report. It will be fixed in v2.2.8+1.20.

**Comment by jakebaldiwn00** [comment]
I'm having the same issue. Just got the mod installed on server and client and I cant craft the elevator block!
